Sources/Sentry/SentryTracer.m

```diff
@@ -278,20 +278,85 @@ - (void)cancelIdleTimeout
 
 - (void)spanFinished:(id<SentrySpan>)finishedSpan
 {
-    SENTRY_LOG_DEBUG(@"Finished span %@", finishedSpan.context.spanId.sentrySpanIdString);
+    SENTRY_LOG_DEBUG(@"Finished span %@", finishedSpan.spanId.sentrySpanIdString);
     // Calling canBeFinished on the rootSpan would end up in an endless loop because canBeFinished
     // calls finish on the rootSpan.
     if (finishedSpan == self.rootSpan) {
-        SENTRY_LOG_DEBUG(@"Cannot call finish on root span with id %@",
-            finishedSpan.context.spanId.sentrySpanIdString);
+        SENTRY_LOG_DEBUG(
+            @"Cannot call finish on root span with id %@", finishedSpan.spanId.sentrySpanIdString);
         return;
     }
     [self canBeFinished];
 }
 
-- (SentrySpanContext *)context
+- (SentryId *)traceId
+{
+    return self.rootSpan.traceId;
+}
```

Comment:
Because we already expose `rootSpan` in `SentryTracer.h`, I don't think we should provide all these passthrough methods. The callsites should instead get this from tracer.rootSpan, it'll make it more obvious what's happening in all those parts of code.

I know this is probably a pain, but I think it's worthwhile to keep the concepts of `tracer` and `span` as decoupled as possible, instead of melding them together any further than they already are.

Reply:
SentryTracer implements SentrySpan protocol, and is the actual return of a `[SentrySDK startTransaction]`, therefore we need this properties to be exposed.
I guess we could make SentryTracer a subclass of a SentrySpan, then we dont need to reimplement all this properties.

Reply:
> I know this is probably a pain, but I think it's worthwhile to keep the concepts of tracer and span as decoupled as possible, instead of melding them together any further than they already are.

I fully agree, but I think that's a different discussion, and we don't need to solve this in this PR. We could do that refactoring later on. Tracer is not public.